README.md

```diff
@@ -16,11 +16,11 @@ This repository contains the markup needed to deploy the floeproject.org website
 News Item Template
```

Comment:
@jhung I reversed the order of the figcaptions and the images in order to provide aria-details at the top of the block of markup - I wasn't sure how to avoid this. I'm concerned though that the captions should come first for accessibility (describing the image before landing on the image). I'm also not sure if the aria-details are working properly - I can't get V/O to read them at all.